applications/CompressiblePotentialFlowApplication/tests/cpp_tests/test_compressible_potential_element.cpp

```diff
@@ -72,5 +72,108 @@ namespace Kratos {
 			KRATOS_CHECK_NEAR(RHS(1), 0.0, 1e-7);
 			KRATOS_CHECK_NEAR(RHS(2), -0.5, 1e-7);
 		}
+
+		/** Checks the CompressiblePotentialFlowElement element.
+	     * Checks the EquationIdVector.
+	     */
+	    KRATOS_TEST_CASE_IN_SUITE(CompressiblePotentialFlowElement_EquationIdVector, CompressiblePotentialApplicationFastSuite)
+		{
+
+			ModelPart modelPart("Main");
+
+			// Variables addition
+			modelPart.AddNodalSolutionStepVariable(POSITIVE_FACE_PRESSURE);		
+
+			// Set the element properties
+			Properties::Pointer pElemProp = modelPart.pGetProperties(0);
+
+			// Geometry creation
+			modelPart.CreateNewNode(1, 0.0, 0.0, 0.0);
+			modelPart.CreateNewNode(2, 1.0, 0.0, 0.0);
+			modelPart.CreateNewNode(3, 1.0, 1.0, 0.0);
+			std::vector<ModelPart::IndexType> elemNodes {1, 2, 3};
+			modelPart.CreateNewElement("CompressiblePotentialFlowElement2D3N", 1, elemNodes, pElemProp);
+
+			Element::Pointer pElement = modelPart.pGetElement(1);
+
+			for(unsigned int i=0; i<3; i++){
+				pElement->GetGeometry()[i].AddDof(POSITIVE_FACE_PRESSURE);
+			}
+
+			Element::DofsVectorType ElementalDofList;
+			pElement->GetDofList(ElementalDofList,modelPart.GetProcessInfo());
+
+			for (int i = 0; i < 3; i++)
+        	{
+        	    ElementalDofList[i]->SetEquationId(i);	
+        	}
+
+			Element::EquationIdVectorType EquationIdVector;
+
+			pElement->EquationIdVector(EquationIdVector, modelPart.GetProcessInfo());
+
+			// Check the EquationIdVector values
+			KRATOS_CHECK_NEAR(EquationIdVector[0], 0, 1e-7);
```

Comment:
Better to use equality check for integer types: `KRATOS_CHECK(EquationIdVector[0] == 0)`.